content/sensu-core/2.0/reference/entities.md

```diff
@@ -4,82 +4,355 @@ description: "The entities reference guide."
-Platform:               darwin
-Platform Family:        
-Platform Version:       10.13.3
-Auto-Deregistration:    false
-Deregistration Handler:
-{{< /highlight >}}
+extended_attributes | 
+-------------|------ 
+description  | Custom attributes to include as with the entity, that appear as outer-level attributes.
+required     | false 
+type         | byte 
+example      | {{< highlight shell >}}"team":"ops"{{</ highlight >}}
 
-### Management
+redact       | 
+-------------|------ 
+description  | List of items to redact from log messages. 
+required     | false 
+type         | array 
+default      | ["password", "passwd", "pass", "api_key", "api_token", "access_key", "secret_key", "private_key", "secret"]
+example      | {{< highlight json >}}"redact": [
+  "extra_secret_tokens"
+]
+{{</ highlight >}}
 
-Entities can be updated interactively through the CLI.
+### System Attributes
 
-{{< highlight shell >}}
-sensuctl entity update entity.localhost
-{{< /highlight >}}
+hostname     | 
+-------------|------ 
+description  | The hostname of the entity. 
+required     | false 
+type         | string 
+example      | {{< highlight shell >}}"hostname": "example-hostname" {{</ highlight >}}
+
+os           | 
+-------------|------ 
+description  | The entity's operating system. 
+required     | false 
+type         | string 
+example      | {{< highlight shell >}}"os": "linux" {{</ highlight >}}
+
+platform     | 
+-------------|------ 
+description  | The entity's operating system distribution. 
+required     | false 
+type         | string 
+example      | {{< highlight shell >}}"platform": "ubuntu" {{</ highlight >}}
+
+platform_family     | 
+-------------|------ 
+description  | The entity's operating system family. 
+required     | false 
+type         | string 
+example      | {{< highlight shell >}}"platform_family": "debian" {{</ highlight >}}
+
+platform_version     | 
+-------------|------ 
+description  | The entity's operating system version. 
+required     | false 
+type         | string 
+example      | {{< highlight shell >}}"platform_version": "16.04" {{</ highlight >}}
 
-To delete an existing entity, simply pass the name of the entity to the `delete` command.
+network     | 
+-------------|------ 
+description  | The entity's network interface list. 
+required     | false 
+type         | [Network][3] 
+example      | {{< highlight json >}}
+"network": {
+      "interfaces": [
+        {
+          "name": "lo",
+          "addresses": [
+            "127.0.0.1/8",
+            "::1/128"
+          ]
+        },
+        {
+          "name": "eth0",
+          "mac": "52:54:00:20:1b:3c",
+          "addresses": [
+            "93.184.216.34/24",
+            "2606:2800:220:1:248:1893:25c8:1946/10"
+          ]
+        }
+      ]
+}
+{{</ highlight >}}
 
-{{< highlight shell >}}
-sensuctl entity delete entity.localhost
+arch         | 
+-------------|------ 
+description  | The entity's system architecture. 
```

Comment:
We should probably specify that we use the binary architecture (`runtime.GOARCH`) to determine the system architecture, meaning if an `amd64` system runs the `386` binary, it will be reported as `386` in Sensu!

For reference: https://github.com/sensu/sensu-go/issues/154

Reply:
👍